scripts/validate-deployments.ts

```diff
@@ -301,6 +310,130 @@ async function validateDeployments(network: string) {
             `${network} (${[...auctioneerMetadata['auction-resolvers'], ...auctioneerMetadata['auction-cops']]}) Api3ServerV1OevExtension auctioneer role statuses are (${goFetchApi3ServerV1OevExtensionAuctioneerRoleStatus.data.map((auctioneerRoleStatus) => (auctioneerRoleStatus === ethers.ZeroHash ? false : true))})`
           );
         }
+
+        // Validate that collateral rate proxy is set
+        const { address: oevAuctionHouseAddress, abi: oevAuctionHouseAbi } = JSON.parse(
+          fs.readFileSync(join('deployments', network, `OevAuctionHouse.json`), 'utf8')
+        );
+        const oevAuctionHouse = new ethers.Contract(
+          oevAuctionHouseAddress,
+          oevAuctionHouseAbi,
+          provider
+        ) as unknown as OevAuctionHouse;
+
+        const goFetchCollateralRateProxyAddress = await go(
+          async () => oevAuctionHouse.collateralRateProxy(),
+          goAsyncOptions
+        );
+        if (!goFetchCollateralRateProxyAddress.success) {
+          throw new Error('OevAuctionHouse collateral rate proxy address could not be fetched');
+        }
+        const oevAuctionChainId = CHAINS.find((chain: any) => chain.alias === network)!.id;
```

Comment:
`@api3/chains` package exports `Chain` type. Avoiding `any` type may be considered

Reply:
Yeah I was being lazy